srcpkgs/bazel/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'bazel'
+pkgname=bazel
+version=3.5.0
+revision=1
+archs="x86_64"
+wrksrc="."
+hostmakedepends="gcc openjdk11 unzip which zip"
+makedepends="python3-devel"
+short_desc="Open-source build and test tool similar to Make, Maven, and Gradle"
```

Comment:
I would use the description from the repository "fast, scalable, multi-language and extensible build system"